lib/exchange.dart

```diff
@@ -47,6 +47,7 @@ class _ExchangeWidgetState extends State<ExchangeWidget> {
   Timer? _updateTimer;
   String _lastAmount = '';
   String _lastReceive = '';
+  Map<String, bool> _isSelected = {"MIN": false, "HALF": false, "MAX": false};
```

Comment:
this should probably be changed to an enum like `enum AmountSliderSelected { none, min, half, max}`